### PR TITLE
Fix grammar in active_record/counter_cache.rb [ci skip]

### PR DESCRIPTION
**Pull request by kenta-s**

### CODE REVIEW

File: activerecord/lib/active_record/counter_cache.rb [hunk 1]
```diff
@@ -18,11 +18,11 @@ module ClassMethods
       #
       # ==== Examples
       #
-      #   # For Post with id #1 records reset the comments_count
+      #   # For the Post with id #1 record, reset the comments_count
```
Comment (maclover7) [inline activerecord/lib/active_record/counter_cache.rb]: I think we can remove the `record`, based on reading seems clear that "record" is implied.

File: activerecord/lib/active_record/counter_cache.rb [hunk 2]
```diff
@@ -18,11 +18,11 @@ module ClassMethods
...
       #   Post.reset_counters(1, :comments)
       #
-      #   # For Post with id #1 records reset the comments_count
-      #   # and updates the +updated_at+ and/or +updated_on+ attributes.
+      #   # For the Post with id #1 record, reset the comments_count
```
Comment (maclover7) [inline activerecord/lib/active_record/counter_cache.rb]: Same as above, I think we can remove "record"

**Review by maclover7 — CHANGES_REQUESTED** [review]
One small change, and then looks good to me!

**Comment by kenta-s** [comment]
@maclover7 Thank you for the comment :) done

**Pull request merged**